tensorboard/program.py

```diff
@@ -220,6 +220,9 @@ def main(self, ignored_argv=('',)):
       event_file = os.path.expanduser(self.flags.event_file)
       efi.inspect(self.flags.logdir, event_file, self.flags.tag)
       return 0
+    if self.flags.version:
```

Comment:
You'll need to define the flag in `core_plugin.py`.  See the `--inspect` flag definition for an example:
https://github.com/tensorflow/tensorboard/blob/dc71eeea403eb027612422ab6cd3061d4f9a10f3/tensorboard/plugins/core/core_plugin.py#L363-L378

Reply:
shashvatshahi1998 if you're not working on this can i send pr for the same??

Reply:
#2100 

@nfelt  please correct me if i am wrong!

Reply:
#2100

Reply:
@nfelt Defined the flag in `core_plugin.py`

See #2100

Reply:
@aman592 I was also working on the issue man.

Reply:
@amanp592 please let @shashvatshahi1998 finish working on this PR since they've already started it, and the functionality should all be submitted as a single PR.

If you're looking for a simple issue, you could pick up https://github.com/tensorflow/tensorboard/issues/1176 which is for ensuring that if --path_prefix is used, the URL printed out includes a trailing slash.  The fix for that would be changing `get_url()` here to add a trailing slash to the URL if there isn't one already there:
https://github.com/tensorflow/tensorboard/blob/dc71eeea403eb027612422ab6cd3061d4f9a10f3/tensorboard/program.py#L519-L527

A PR for that should also include a test to confirm the bug is fixed, which would go in https://github.com/tensorflow/tensorboard/blob/dc71eeea403eb027612422ab6cd3061d4f9a10f3/tensorboard/program_test.py#L43

Reply:
@shashvatshahi1998 ohk cool sorry for that!

Reply:
@nfelt Thankyou for refering me #1176 i'll look what can be done

Reply:
@amanp592 no worries